policies/coding-style.md

```diff
@@ -232,6 +244,13 @@ can optionally strip the trailing whitespace; however, if applying
 a series of patches, this may make later patches in the series fail by
 changing their context lines.
 
+Avoid empty lines at the beginning or at the end of a file.
+
+Avoid multiple empty lines in a row.
+
+Have an empty line between variable declarations and subsequent statements.
+Do not mix variable declarations and statements.
```

Comment:
Not sure this rule really fits in the "Spaces" chapter. Although I'm not sure where else you would put it other than "miscellaneous".

Reply:
Well, right, the placing is not ideal here.
I also was unsure where else to put the sentence
```
 Do not mix variable declarations and statements. 
```
Likely best in the Functions chapter?

Reply:
That's not quite right either (although probably better than "Spaces"). Possibly "Naming"...i.e. it's a rule about where to name new variables!

Reply:
Hmm, that sentence is not really about names but about the order of variable decls and statements.
It is relevant for function bodies, and there only.
And since moreover there is no chapter specifically for variable decls nor for statements,
I'm meanwhile pretty sure that the functions chapter is the best fit.

Reply:
I agree, functions is the best fit.

Reply:
@mattcaswell, is this ok by you now?

Reply:
Yes - it's not perfect since it applies to any code block (not just at the start of a function). But I can live with it.